nixos/modules/services/networking/biboumi.nix

```
@@ -17,6 +17,13 @@ in
     services.biboumi = {
       enable = mkEnableOption (lib.mdDoc "the Biboumi XMPP gateway to IRC");
 
+      package = mkOption {
+        description = lib.mdDoc "The Biboumi package to use";
+        default = pkgs.biboumi;
+        defaultText = literalExpression "pkgs.biboumi";
+        type = types.package;
+      };
```

```suggestion
      package = mkPackageOption pkgs "biboumi" { };
```
otherwise lgtm. thanks!